dashboard/app/controllers/projects_controller.rb

```diff
@@ -215,6 +216,14 @@ def show
     if params[:key] == 'artist'
       @project_image = CDO.studio_url "/v3/files/#{@view_options['channel']}/_share_image.png", 'https:'
     end
+    FirehoseClient.instance.put_record(
+      'projects-events',
```

Comment:
@joannepolsky can you comment on whether this new event type will successfully result in the record appearing in redshift? or is our firehose + redshift integration currently only configured for events of type 'analysis-events' as we are already recording at https://github.com/code-dot-org/code-dot-org/blob/de8188c6dc1e1b6f2c5c3ad3e8bc0d2f7e30fa80/dashboard/app/models/studio_person.rb#L43-L56 ?

Reply:
This actually was a problem, we got a spew of Honeybadger alerts coming in at https://app.honeybadger.io/projects/3240/faults/34958824#notice-summary

As a workaround for now, I created a `projects-events` Firehose stream (https://console.aws.amazon.com/firehose/home?region=us-east-1#/details/projects-events?edit=false), and pointed it at an S3 bucket, which is a bit simpler to do than get the data all the way into Redshift. We'll either want to change that over at some point so that the data gets to the database (Ben might need to create another table for us), or change the code to refer to the existing table.

@joannepolsky and @wjordan , Asher and I definitely intended to do another pass over this implementation to tighten things up and make it a bit easier to test. It probably makes sense to either hard-code the specific stream we're targeting into the FirehoseClient or make sure we have pretty clear documentation on the existing streams, since it's a bit of work to set up a new one.

Reply:
CC @asherkach.